Cannot create second Asciidoctor task in 1.5.0

The following build script is enough to recreate the issue

``` groovy
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.0'
    }
}

apply plugin: 'org.asciidoctor.gradle.asciidoctor'
import org.asciidoctor.gradle.AsciidoctorTask
task secondDoc( type : AsciidoctorTask)
```

Running `gradle secondDoc` results in 

```
:secondDoc FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':secondDoc'.
> java.lang.ClassNotFoundException: org.asciidoctor.Asciidoctor$Factory

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Problem occurs irrespective of whether task is configured or not.

--- a/src/main/groovy/org/asciidoctor/gradle/AsciidoctorTask.groovy
+++ b/src/main/groovy/org/asciidoctor/gradle/AsciidoctorTask.groovy
@@ -162,6 +162,9 @@ class AsciidoctorTask extends DefaultTask {
 
     @TaskAction
     void processAsciidocSources() {
+        if(classpath == null) {
+            classpath = project.configurations.getByName(AsciidoctorPlugin.ASCIIDOCTOR)
+        }
         validateInputs()
         outputDir.mkdirs()
 

--- a/src/test/groovy/org/asciidoctor/gradle/AsciidoctorTaskSpec.groovy
+++ b/src/test/groovy/org/asciidoctor/gradle/AsciidoctorTaskSpec.groovy
@@ -50,6 +50,7 @@ class AsciidoctorTaskSpec extends Specification {
 
     def setup() {
         project = ProjectBuilder.builder().withName('test').build()
+        project.configurations.create(ASCIIDOCTOR)
         mockAsciidoctor = Mock(AsciidoctorProxy)
         testRootDir = new File('.')
         srcDir = new File(testRootDir, ASCIIDOC_RESOURCES_DIR)
@@ -600,4 +601,5 @@ class AsciidoctorTaskSpec extends Specification {
         then:
             1 * mockAsciidoctor.renderFile(_, _)
     }
+
 }

